### PR TITLE
app extension for other apps added

### PR DESCRIPTION
**Pull request by aman-alfresco**

### CODE REVIEW

File: app/src/main/java/com/alfresco/content/app/activity/ExtensionActivity.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    private lateinit var actionBarController: ActionBarController
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_extension)
+
+        when (intent?.action) {
+            Intent.ACTION_SEND -> {
+                println("EXTENSION SINGLE IMAGE")
+                if (intent.type?.startsWith("image/") == true) {
```
Comment (ankitgoyalgloballogic) [inline app/src/main/java/com/alfresco/content/app/activity/ExtensionActivity.kt:31]: It should handle multiple types of files like images, video, pdf etc?

File: shareextension/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
```
Comment (ankitgoyalgloballogic) [inline shareextension/src/main/res/values/colors.xml:3]: It should be removed

**Review by ankitgoyalgloballogic — APPROVED** [review]

File: app/src/main/java/com/alfresco/content/app/activity/ExtensionActivity.kt (hunk 1 above)
Comment (aman-alfresco) [reply app/src/main/java/com/alfresco/content/app/activity/ExtensionActivity.kt:31]: I've not worked on that part yet so will do it later along with the planned story.

File: shareextension/src/main/res/values/colors.xml (hunk 2 above)
Comment (aman-alfresco) [reply shareextension/src/main/res/values/colors.xml:3]: It will be updated in the next PR

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**